### PR TITLE
Added setMetadataStream to TorusStorageLayer

### PR DESCRIPTION
**Pull request by rathishubham7**

### CODE REVIEW

File: packages/default/src/index.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ class ThresholdKey extends TKey {
       finalServiceProvider = serviceProvider;
     }
     if (!storageLayer) {
-      finalStorageLayer = new TorusStorageLayer({ serviceProvider: finalServiceProvider, hostUrl: "https://metadata.tor.us" });
+      finalStorageLayer = new TorusStorageLayer({ serviceProvider: finalServiceProvider, hostUrl: "http://localhost:5051" });
```
Comment (rathishubham7) [inline packages/default/src/index.ts:23]: revert this

File: packages/default/src/index.ts [hunk 2]
```diff
@@ -41,7 +41,7 @@ class ThresholdKey extends TKey {
...
```
Comment (rathishubham7) [inline packages/default/src/index.ts:44]: revert this

**Review by YZhenY — APPROVED** [review]
lgtm

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**